security/src/main/java/io/micronaut/security/authentication/AuthenticationExceptionHandler.java

```diff
@@ -40,15 +44,41 @@
 public class AuthenticationExceptionHandler implements ExceptionHandler<AuthenticationException, MutableHttpResponse<?>> {
     protected final ApplicationEventPublisher eventPublisher;
 
+    private final ErrorResponseProcessor<?> responseProcessor;
+
+    /**
+     * @param eventPublisher The event publisher
+     */
     public AuthenticationExceptionHandler(ApplicationEventPublisher eventPublisher) {
+        this(eventPublisher, null);
```

Comment:
Instead of `null` can you manually instantiate `HateosErrorResponseProcessor` which is the default implementation of `ErrorResponseProcessor`?